sources/lib/util/options.ml

```diff
@@ -132,12 +132,15 @@ let vno_backward = ref false
 let vno_sat_learning = ref false
 let vinstantiate_after_backjump = ref false
 let vdisable_weaks = ref false
-let vinput_format = ref "why"
 let vanswers_with_loc = ref true
 
-type output = ONative | OSmtlib | OSZS
+type input = INative | ISmtlib | IWhy3 (* | ISZS *)
```

Comment:
Since disambiguation of adt with OCaml, prefixing constructors name is not neded nor really useful, and I find it particularly annoying, so I'd much rpefer removing the first `I` (and simialry for the `O` in output formats).

Reply:
Additionally, it might be useful to have all type definitions at the beginning of a module rather than spread all around a file ?

Reply:
Also, I'd rather name this `input_format` to distinguish it from a representation of inputs file/pb (which might be : `type input = File of string | Stdin` for instance).